api/v1alpha1/warehouse_types.go

```diff
@@ -249,8 +249,7 @@ type ImageSubscription struct {
 	// constraints, which means the latest semantically tagged version of an image
 	// will always be used. Care should be taken with leaving this field
 	// unspecified, as it can lead to the unanticipated rollout of breaking
-	// changes. Refer to Image Updater documentation for more details.
-	// More info: https://github.com/masterminds/semver#checking-version-constraints
+	// changes.
```

Comment:
This may have been poorly worded by me initially and it may have caused you some confusion.

Image Updater and masterminds/semver are two different things.

We _do_ want to remove anything that refers readers to Image Updater for more info because we're no longer using Image Updater anywhere in Kargo.

masterminds/semver is a library we use for parsing and comparing semantic version strings. (Image Updater happens to use it as well. It's a popular library.)

At any rate... we should _keep_ references to masterminds/semver docs as they're still the definitive source of information for how version constraints work.

Reply:
This applies throughout the PR.